src/usage/schema.md

```diff
@@ -0,0 +1,9 @@
+# Schema
+
+Did you land here because of cryptic errors? Then please install the following official GitHub Actions VSCode Extension https://marketplace.visualstudio.com/items?itemName=GitHub.vscode-github-actions and review all errors/warnings shown by it.
+
+
+Once you are sure this is an error in act, please look for existing issues or visit https://github.com/nektos/act/issues/new/choose to report a new issue.
+
+_Act's built-in schema validation errors are not easy to understand while keeping the development effort reasonable for open source_

```

Comment:
```suggestion
_Act's built-in schema validation errors are not easy to understand while keeping the development effort reasonable for open source._
```

I'm not entirely certain what this sentence is trying to say, but I'm assuming it's a sentence.

Reply:
I've tried contributing to a bunch of the actions/* repositories, a number of them are indeed wastelands.